docs/components/FileInputView.jsx

```diff
@@ -170,6 +170,13 @@ export default class FileInputView extends React.PureComponent {
               optional: true,
               defaultValue: <code>FormElementSize.FULL_WIDTH</code>,
             },
+            {
+              name: "isMessagingAttachmentInput",
+              type: "boolean",
+              description:
+                "Optional boolean which dictates if this FileInput is being used for uploading message attachments. If so, the component is rendered specifically for that purpose",
+              optional: true,
+            },
```

Comment:
Wondering if we should avoid adding this to the docs for FileInput since we mostly use this prop internally from within MessagingInput